src/ratekeeper.py

```diff
@@ -341,13 +344,24 @@ def update_DRVBusy(self, mcc):
                         m_status = movc.status(self.mover_to, self.mover_retries)
                         if e_errors.is_ok(m_status):
                             valid_drives.append(m_status['media_changer_device'])
-
+                            # find in drives list
+                            for drive in drives_list:
+                                if drive['name'] == m_status['media_changer_device']:
+                                    if not drive['type']:
```

Comment:
suggest to leave a comment why drive['type'] can be None and why the lookup needs to be done. 
As I understand lack of drive type when querying TFF1 broke the plotting of active drives.

Reply:
I do not know why. This was done according to your comments that the drive type was ''. My impression was that it affects plots. So I do not know what comment to put here.

Reply:
I think you misunderstood my comment.  I wanted o have this 

```
 if not drive['type']:
....
```

bit commented. "That TFF1 does not have a type defined and this is how we get that type in in this case" or something along these lines if above is not correctly describing  why drive['type'] == None